docs/source/dev_install.md

```diff
@@ -15,33 +15,18 @@ available in the PATH used with sudo.
 
         git clone https://github.com/ipython/ipywidgets
 
-1. Build the Jupyter Widgets package
-
-        cd jupyter-js-widgets
-        npm install
-        cd ..
-
-2. Install the Jupyter Widgets nbextension
-
-        cd widgetsnbextension
-        npm install
-        npm run update:widgets
-        pip install -v -e .
-        cd ..
-        jupyter nbextension install --py --symlink --user widgetsnbextension
-        jupyter nbextension enable --py --user widgetsnbextension
-
-3. Dev-install of the package (run from ipywidgets repo directory):
+1. Navigate into the cloned repo and install
 
         cd ipywidgets
-        pip install -v -e .
+        bash dev-install.sh --user
```

Comment:
I think that's what we were already doing.  I don't really have an opinion on the mater, other than --user has always worked well for me.